CarRentals/CarRentals/Controllers/CarCRUD.cs

```diff
@@ -18,24 +18,20 @@ public class CarCRUD
             Converters = { new JsonStringEnumConverter() }
         };
 
-
         private List<Car> CarList;
 
-
         public CarCRUD()
         {
             ReadJson();
         }
 
-
-      
         public void Create(Car car)
         {
-            CarList.Add(car);
+            if(car != null)
+                CarList.Add(car);
             SaveChanges();
         }
 
-
         public Car Get(int id)
```

Comment:
throwing exceptions is expensive, we should be really picky about when and why. not finding a car may not be enough to throw one, perhaps we can just return null? what do you think?

Reply:
Thanks! I have changed all this lines in only one, deleting exceptions and putting a LINQ lambda expression.